WooCommerce/Classes/ViewRelated/Dashboard/DashboardView.swift

```diff
@@ -178,9 +178,47 @@ private extension DashboardView {
                     }
                 }
             }
+
+            if canShowShareCard {
+                shareStoreCard
+            }
         }
     }
 
+    var shareStoreCard: some View {
+        VStack {
+            Image(uiImage: .blazeSuccessImage)
+                .padding(Layout.imagePadding)
+
+            Text(Localization.ShareStoreCard.title)
+                .headlineStyle()
+                .multilineTextAlignment(.center)
+
+            Text(Localization.ShareStoreCard.subtitle)
+                .bodyStyle()
+                .multilineTextAlignment(.center)
+                .padding(.horizontal, Layout.elementPadding)
+                .padding(.top, Layout.textPadding)
+
+            if let url = viewModel.siteURLToShare {
+                ShareLink(item: url) {
+                    Text(Localization.ShareStoreCard.shareButtonLabel)
+                }
+                .buttonStyle(PrimaryButtonStyle())
+                .padding(.horizontal, Layout.elementPadding)
+                .padding(.vertical, Layout.elementPadding)
+            }
+        }
+        .background(Color(.clear))
+        .cornerRadius(Layout.cornerRadius)
```

Comment:
This is handled in https://github.com/woocommerce/woocommerce-ios/pull/12521/commits/2d25991531281f1d75a42f6cdad32984f8c55d80

Reply:
I see that you removed the background color instead of the corner radius. It should be the other way around right?

Reply:
Oh! I misunderstood what you meant because as it turned out `.background(Color(.clear))` is also not needed. Both background color and corner radius are gone now in https://github.com/woocommerce/woocommerce-ios/pull/12521/commits/32fbedeefe94e4bf4becd2b76f51a9d9a8b38c7f